src/app/CodeRepo/lib/types/revision.js

```diff
@@ -198,12 +202,12 @@ class Revision extends Type {
                     _ref: true
                 };
             } else if (users.length === 0) {
-                ServiceMgr.instance.log("info", `Found no user with email ${userEmail}`);
+                ServiceMgr.instance.log("info", `Found no user matching query '${JSON.stringify(query)}'`);
             } else {
-                ServiceMgr.instance.log("error", `Found ${users.length} users with email ${userEmail}`);
+                ServiceMgr.instance.log("error", `Found ${users.length} users matching query '${JSON.stringify(query)}'`);
             }
         } catch (error) {
-            ServiceMgr.instance.log("error", `Can't get user ref for user with email ${userEmail}`, error);
+            ServiceMgr.instance.log("error", `Error when executing query '${JSON.stringify(query)}'`, error);
```

Comment:
Write more describing error. It shall say when it did go wrong?
"Failed to find user matching query"?